wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -24,72 +24,73 @@ module Pos.Wallet.Web.Methods.Logic
+
+import           Pos.Aeson.ClientTypes        ()
+import           Pos.Aeson.WalletBackup       ()
+import           Pos.Core                     (Address, Coin, mkCoin, sumCoins,
+                                               unsafeIntegerToCoin)
+import           Pos.Crypto                   (PassPhrase, changeEncPassphrase,
+                                               checkPassMatches, emptyPassphrase)
+import           Pos.Txp                      (applyUtxoModToAddrCoinMap)
+import           Pos.Util                     (maybeThrow)
+import qualified Pos.Util.Modifier            as MM
+import           Pos.Util.Servant             (encodeCType)
+import           Pos.Wallet.KeyStorage        (addSecretKey, deleteSecretKey,
+                                               getSecretKeysPlain)
+import           Pos.Wallet.Web.Account       (AddrGenSeed, genUniqueAccountId,
+                                               genUniqueAddress, getAddrIdx, getSKById)
+import           Pos.Wallet.Web.ClientTypes   (AccountId (..), CAccount (..),
+                                               CAccountInit (..), CAccountMeta (..),
+                                               CAddress (..), CId, CWAddressMeta (..),
+                                               CWallet (..), CWalletMeta (..), Wal,
+                                               addrMetaToAccount, encToCId, mkCCoin)
+import           Pos.Wallet.Web.Error         (WalletError (..))
+import           Pos.Wallet.Web.Mode          (MonadWalletWebMode)
+import           Pos.Wallet.Web.State         (AddressLookupMode (Existing),
+                                               CustomAddressType (ChangeAddr, UsedAddr),
+                                               addWAddress, createAccount, createWallet,
+                                               getAccountIds, getAccountMeta,
+                                               getWalletAddresses,
+                                               getWalletMetaIncludeUnready,
+                                               getWalletPassLU, isCustomAddress,
+                                               removeAccount, removeHistoryCache,
+                                               removeTxMetas, removeWallet,
+                                               setAccountMeta, setWalletMeta,
+                                               setWalletPassLU, setWalletReady)
+import qualified Pos.Wallet.Web.State         as WS
+import           Pos.Wallet.Web.State.Storage (WalBalancesAndUtxo)
+import           Pos.Wallet.Web.Tracking      (CAccModifier (..), CachedCAccModifier,
+                                               fixCachedAccModifierFor,
+                                               fixingCachedAccModifier, sortedInsertions)
+import           Pos.Wallet.Web.Util          (decodeCTypeOrFail, getAccountAddrsOrThrow,
+                                               getWalletAccountIds)
```

Comment:
No large imports whitespace changes in this branch please, just change the lines that need to change.